index.html

```diff
@@ -793,14 +793,19 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
       <ol>
         <li><p>If <var>track</var>.{{MediaStreamTrack/muted}} is <code>true</code>, wait for <var>track</var>.{{MediaStreamTrack/muted}}
           to become <code>false</code> or <var>track</var>.{{MediaStreamTrack/readyState}} to be "ended".</p></li>
-        <li><p>If <var>track</var>.{{MediaStreamTrack/readyState}} is "ended", abort these steps.</p></li>
-        <li><p>If <var>track</var>'s capabilities, constraints and settings are matching <var>source</var> configuration, abort these steps.
-        <li>
-          <!-- FIXME: Export capabilities, constraints and settings so that we can use them here. -->
-          <p>Update <var>track</var>'s capabilities, constraints and settings according <var>track</var>'s underlying source.</p>
-        </li>
-        <li>
-          <p>[=Fire an event=] named <var>configurationchange</var> on <var>track</var>.</p>
+        <li><p>[=Queue a task=] on [=current settings object=]'s [=environment settings object/responsible event loop=] 
+          to perform the following steps:</p>
```

Comment:
We could add a note stating that we waited for the track to be muted=false and the task will be run before any other task that might have put the track to muted=true.

Reply:
Done